PowerEditor/installer/nativeLang/estonian.xml

```diff
@@ -1,903 +1,1008 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <!--
-*Estonian language file for Notepad++*
-
-	** Created By:- Yusuf Mullaji *
-		Email Id-> yusufmullaji@rocketmail.com
-
+Estonian translation by Andres Traks
+https://github.com/AndresTraks/
```

Comment:
i am not satisfying my ego i just expected for my name to be mentioned as contributor. Though, its your decision and i will support it. Thank You.